packages/clerk-js/src/core/auth/AuthCookieService.ts

```diff
@@ -110,27 +110,47 @@ export class AuthCookieService {
     this.poller.startPollingForSessionToken(() => this.refreshSessionToken());
   }
 
-  private refreshTokenOnVisibilityChange() {
-    document.addEventListener('visibilitychange', () => {
+  private refreshTokenOnFocus() {
+    window.addEventListener('focus', () => {
```

Comment:
We should either replace this event listener call here with a `PageLifecycle`  helper or ditch the `PageLifecycle` 
helper altogether as it feels a little unnecessary right now

Reply:
Agreed, good idea 👍